nexus/resource_repository.go

```diff
@@ -46,7 +46,7 @@ func resourceRepository() *schema.Resource {
 			"apt": {
 				Type:          schema.TypeList,
 				Optional:      true,
-				ConflictsWith: []string{"bower", "docker", "docker_proxy", "maven"},
+				ConflictsWith: []string{"bower", "docker", "docker_proxy", "maven", "helm"},
```

Comment:
There is no "helm" key on top-level of schema, you shouldn't add this token here (or you forgot to add some helm-specific configuration options)